src/pdb/mod.rs

```diff
@@ -495,81 +496,20 @@ impl RowGroup {
     }
 }
 
-#[derive(Debug)]
-/// A string.
-pub enum DeviceSQLString {
-    /// A short ASCII encoded string (max 126 characters).
-    ShortASCII(String),
-    /// A long ASCII encoded string (max 65535 characters).
-    LongASCII(String),
-    /// A long UTF-16 little-endian encoded string (max 32767 characters).
-    LongUTF16LE(String),
-    /// A ShortASCII variant that is needed to workaround the special ISRC string
-    /// NOTE: the serialized string is a null-terminated C-String
-    ISRCASCII(String),
-}
-
 impl DeviceSQLString {
     fn parse(input: &[u8]) -> IResult<&[u8], DeviceSQLString> {
```

Comment:
I'm not sure I understand. This basically adds another impl with a nom-compatible parse method, so we can use the binrw-based DeviceSQLString type in the context of nom?

Reply:
Correct. This allows removing the old `DeviceSQLImplementation` completely.

This `impl` block is removed in #64, because it's only needed for the transitional phase where the binrw parser still needs nom.

Reply:
Any objections?

Reply:
Nope, thanks for the explanation. I was confused by the disconnected impl block, but if it compiles, it works.

Reply:
Yeah, the `string.rs` module is currently binrw-only, and I wanted to keep it that way, this is why I put the impl block here (which is also the only module where the method is actually used).